<ui version="4.0" >
 <class>SaveViewDialog</class>
 <widget class="QDialog" name="SaveViewDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Save View</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pmchart.qrc" >:/images/view.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="fileNameLabel" >
       <property name="minimumSize" >
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameLineEdit" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QListView" name="dirListView" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flow" >
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize" >
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="pathLabel" >
       <property name="minimumSize" >
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pathComboBox" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>8</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="parentToolButton" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Parent</string>
       </property>
       <property name="statusTip" >
        <string/>
       </property>
       <property name="whatsThis" >
        <string>Open the parent directory</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/go-previous.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>8</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="userToolButton" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/toolusers.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="preserveHostCheckBox" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>Preserve hostnames in View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="preserveSizeCheckBox" >
         <property name="text" >
          <string>Preserve window geometry in View</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="savePushButton" >
         <property name="text" >
          <string>Save</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pmchart.qrc" />
 </resources>
</ui>
